### ISSUE TITLE
Pin defines for the display?

### ISSUE THREAD

**Issue by jpaustin** [fowlerk/ESP8266-Color-Weather-Station-v8#4]
Can you summarize the hardware hookup? I see the display lists only the CS and DC pins. I am assuming the remainder are the standard SPI interface pins of the ESP8266? I don't have the WeMos device but a NODEMCU 1.0 so would be nice to have both shown.

**Comment by fowlerk** [comment]
Yes, these are the standard SPI pins, though my recollection is that there is a pin on the display that can be used to PWM the brightness (called LED?); most hookups just wire this to VCC and don't bother.  I just used squix78's original pinout shown on his Github site.  When I get time, I'll copy and add this to the README, but in the meantime you can just reference it there.  It's been a long time since I've used any NodeMCU boards, and don't know if I can put my hands on one, so you and Google might be on your own for remapping the PIN numbers.

**Comment by fowlerk** [comment]
Okay, I've added a section to the README with the wiring connections.  As I noted, please check/verify the pin references for the NodeMCU; it appears to me to be the same (and as documented on squix78's repo and according to Google), but use at your own risk as I have not tested these.

**Comment by jpaustin** [comment]
Yes the NODEMCU 1.0 (ESP8286) wiring is the same. I verified it with both a Wemos and NODEMCU board and they both worked fine. Only issue was I needed to add 100uF cap at the VDD/GND pins of the display board to eliminate the white screen. Also soldered the bypass jumper (J1) on the back of the display to allow the 3.3V to directly power the display. If not jumpered the 3.3V goes into a voltage regulator (intended for 5V operation) and drops the voltage to the display down to about 3V or so.

**Comment by fowlerk** [comment]
Great, thanks for verifying the NodeMCU wiring.  I intended to do that some day, but too many other projects going on right now.  And thanks for the LCD tips; I'd added a smoothing capacitor, which helped substantially, but did not eliminate the problem totally.  I'll look for the regulator bypass jumper and give that a try when I get a few minutes.
